package-lock.json

```diff
@@ -15038,7 +15038,7 @@
     },
     "into-stream": {
       "version": "3.1.0",
-      "resolved": "http://registry.npmjs.org/into-stream/-/into-stream-3.1.0.tgz",
+      "resolved": "https://registry.npmjs.org/into-stream/-/into-stream-3.1.0.tgz",
```

Comment:
can we remove the package-lock from the change? it doesn't seem related to it.

Reply:
Sure, to give some context to this, every time I make a webapp change, my npm version makes this change. I had been reverting the change since the past few months, so thought of including it this time. I will revert.